xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,428 @@
+     &pace; to manage guest nodes and remote nodes.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+
+ <sect1 xml:id="sec.ha.pmremote.term">
+  <title>Terminology</title>
+  <variablelist>
+   <varlistentry>
+    <term>Pacemaker Remote (&pmremote;)</term>
+    <listitem>
+     <para>
+      A service daemon that makes it possible to use a node as a &pace;
+      node without deploying the full cluster stack. While nodes that run
+      this service may run cluster resources and most
+      commandline tools, they do not support other functions of full cluster
+      nodes. This includes fencing execution, quorum voting, and DC
+      eligibility.
+     </para>
+     <note>
+      <title>Different Naming for Service and Daemon</title>
+      <para>The systemd service of Pacemaker Remote is named &pmremote;
+       whereas the name or the process/daemon is &pmrm_daemon;
+       (a trailing <quote>d</quote> after its name).
+      </para>
+     </note>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Cluster Node</term>
+    <listitem>
+     <para>
+      A node that runs all &pace; components along with the complete
+      high-availability &corosync; stack. Cluster nodes may perform the
+      following tasks:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>Run cluster resources.</para>
+      </listitem>
+      <listitem>
+       <para>Run all command-line tools, such as <command>crm</command>,
+        <command>crm_mon</command>, and others.</para>
+      </listitem>
+      <listitem>
+       <para>Count toward cluster quorum.</para>
+      </listitem>
+      <listitem>
+       <para>Serve as the cluster's designated coordinator (DC).</para>
+      </listitem>
+     </itemizedlist>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Remote Node</term>
+    <listitem>
+     <para>
+      A special resource (ocf:pacemaker:remote) is required to be running on
+      a cluster node for managing communication with the baremetal remote
+      node in that case.
+      <remark>toms 2017-03-29: According to gao-yan: The scenario of baremetal
+       remote node deserves a separate chapter I think.
+      </remark>
+      A physical machine that runs the &pmremote; daemon. Remote nodes
+      includes a special resource (often called <quote>baremetal case</quote>)
+      for managing communication with the cluster.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Guest Node</term>
+    <listitem>
+     <para>
+      A virtual host that runs &pmremote;. Unlike remote nodes,
+      a guest node is a resource managed by the cluster. As such, they are
+      not bond to the 32 node limitation of the cluster stack. However, they
+      they behave as regular cluster nodes.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect1>
+
+  <sect1 xml:id="sec.ha.pmremote.concept">
+   <title>Conceptual Overview</title>
+   <!-- Add a picture? -->
```

Comment:
See http://clusterlabs.org/doc/en-US/Pacemaker/1.1/html-single/Pacemaker_Remote/#_expanding_the_cluster_stack and also if you watch from https://youtu.be/yq5nYPKxBCo?t=26m9s onwards there are several ideas for how to depict it graphically.